Makefile

```diff
@@ -1,20 +1,20 @@
-TS = $(shell date +%Y%m%d-%H%M%S)
+TS := $(shell date +%Y%m%d-%H%M%S)
 
-VERSION = $(shell mvn -f proxy -q -Dexec.executable=echo -Dexec.args='$${project.version}' --non-recursive exec:exec)
+VERSION := $(shell mvn -f proxy -q -Dexec.executable=echo -Dexec.args='$${project.version}' --non-recursive exec:exec)
+ARTIFACT_ID := $(shell mvn -f proxy -q -Dexec.executable=echo -Dexec.args='$${project.artifactId}' --non-recursive exec:exec)
```

Comment:
this take around 5 sec each... I may change it in a future, but is ok for now.

Reply:
Sounds good. Thanks @laullon